elastic/kibana
# [SearchProfiler] Handle scenario when user has no indices

Fixes https://github.com/elastic/kibana/issues/125080

This PR improves the error a user receives if they attempt to run Search Profiler without any data.

Flaky test build - https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/301

### How to test
1. Start up a fresh ES/Kibana with no data.
2. Navigate to Search Profiler. Use the default query and click "Profile".
3. Verify error notification displays.

### Screenshots
<img width="1205" alt="Screen Shot 2022-03-18 at 10 34 28 AM" src="https://user-images.githubusercontent.com/5226211/159022655-bf4fec9a-acfc-46ce-ba19-30cb94bcc101.png">
